src/languages/es.js

```diff
@@ -250,18 +250,19 @@ export default {
         },
     },
     signInPage: {
-        expensifyDotCash: 'Expensify.cash',
-        expensifyIsOpenSource: 'Expensify.cash es open source',
+        expensifyDotCash: 'New Expensify',
```

Comment:
```suggestion
        expensifyDotCash: 'Nuevo Expensify',
```
Right? Since that's what we say elsewhere